package.json

```diff
@@ -32,6 +32,7 @@
     "i18next-browser-languagedetector": "^8.0.0",
     "i18next-http-backend": "^2.5.2",
     "inquirer": "^8.0.0",
+    "jest-watch-typeahead": "^2.2.2",
```

Comment:
Why was the `jest-watch-typeahead` package added? It seems out of scope of the original issue